t/sql.t

```diff
@@ -1,26 +1,35 @@
 use Mojo::Base -strict;
 
 use Test::More;
+use SQL::Abstract::Test import => [qw(is_same_sql_bind) ];
```

Comment:
We don't really use `qw()` for a single element. And this line looks like it never ran through perltidy.